src/segment_set.rs

```diff
@@ -224,6 +225,17 @@ impl SegmentSet {
         out
     }
 
+	/// Returns the commands parsed from the $j comments
+	pub fn parser_commands(&self) -> Vec<(StatementAddress, Command)> {
+        let mut out = Vec::new();
```

Comment:
Inconsistent formatting: this is five spaces, there are some accurrences of two space and four space indentation elsewhere. Probably we should use `rustfmt` to keep a consistent style, assuming it doesn't do major damage to sorear's original formatting.

Reply:
I agree!
I could do a `fmt` run, and even a `clippy` run. They shall even ideally be built in the process, like in the CI or in the push.

Reply:
I would suggest to run `cargo fmt` after this PR is merged, otherwise it will result in even more changes, and make it much more difficult to review.